modules/mpi/tutorials/CMakeLists.txt

```diff
@@ -1,6 +1,10 @@
 ## Copyright 2009-2020 Intel Corporation
 ## SPDX-License-Identifier: Apache-2.0
 
+if(NOT OSPRAY_MPI_BUILD_TUTORIALS)
```

Comment:
This is redundant, because folder `tutorials` is only processed/included if `OSPRAY_MPI_BUILD_TUTORIALS` is on (which additionally is off per default).